### PR TITLE
feat: moving to pipenv

### PR DESCRIPTION
**Pull request by AlexanderSutul**
#191
So I've tried to move the project to another dep system. Check please.

### CODE REVIEW

**Comment by egregors** [comment]
Wow! Looks pretty cool! I'll check this out ASAP, thanks a lot 🔥 🔥 🔥 

tip: green builds makes things easier ;)

File: compose/dev/Dockerfile [hunk 1]
```diff
@@ -3,14 +3,12 @@ FROM python:3.9-slim
...
 ENV PYTHONPATH /app
-ENV POETRY_VERSION=1.1.4
 
 COPY . /app
 WORKDIR /app
 
-RUN pip install "poetry==$POETRY_VERSION"
-RUN poetry config virtualenvs.create false
-RUN poetry install
+RUN pip install pipenv
```
Comment (C-Pro) [inline compose/dev/Dockerfile:10]: I recommend to specify the version here, or we'll eventually get broken build suddenly

**Review by C-Pro — APPROVED** [review]
👍 fix the build

File: compose/dev/Dockerfile (hunk 1 above)
Comment (AlexanderSutul) [reply compose/dev/Dockerfile:10]: Good idea. I'm going to fix these comments this weekend 🚀 
Thanks for the advice.

Comment (egregors) [reply compose/dev/Dockerfile]: GJ

**Review by egregors — APPROVED** [review]

**Pull request merged**